spec/queries/district_analytics_query_spec.rb

```diff
@@ -120,8 +82,11 @@
             total_registered_patients: 0
           }
         }
+        refresh_views
 
-        expect(analytics.call).to eq(expected_result)
+        with_reporting_time_zone do
```

Comment:
We need to create the data in UTC (the default), and grab analytics in reporting tz...hence the addition of the block helper here.  Without this we will get month boundary issues.